openedx/features/survey_report/admin.py

```diff
@@ -0,0 +1,52 @@
+"""
+Django Admin page for SurveyReport.
+"""
+
+
+from django.contrib import admin
+from .models import SurveyReport
+
+
+class SurveyReportAdmin(admin.ModelAdmin):
+    """
+    Admin to manage survey reports.
+    """
+    readonly_fields = (
+        'courses_offered', 'learners', 'registered_learners',
+        'enrollments', 'generated_certificates', 'extra_data',
+        'created_at'
+    )
+
+    list_display = (
+        'id', 'summary', 'created_at'
+    )
+
+    def summary(self, obj) -> str:
+        """
+        Show a summary of the survey report.
+        info:
+        - Courses: Total number of active unique courses.
+        - Learners: Recently active users with login in some weeks.
+        - Enrollments: Total number of active enrollments in the platform.
+        """
+        return f"Courses: {obj.courses_offered}, Learners: {obj.learners}, Enrollments: {obj.enrollments}"
+
+    def has_add_permission(self, request):
+        """
+        Removes the "add" button from admin.
+        """
+        return False
+
+    def changeform_view(self, request, object_id=None, form_url='', extra_context=None):
+        """
+        Removes the "save" buttons from admin change view.
+        """
+        extra_context = extra_context or {}
+
+        extra_context['show_save'] = False
+        extra_context['show_save_and_continue'] = False
+        extra_context['show_save_and_add_another'] = False
```

Comment:
is it ok to remove reports? I ask for audit reasons

Reply:
I don't find a reason to remove the delete button, after sent the report, I think that the user should not be forced to save the report if he doesn't want to.

@felipemontoya what do you think about this?

Reply:
This one is tricky. Although I agree that we should not force the users to save the report if they don't want, the sent report will not be deleted, and the local delete option could give the false impression that the remote report was also deleted. I would also agree that it is better not to leave the option of deleting. At least if the report is already sent.

I don't think that having a simple record on the DB is such a big issue for anyone. And If they absolutely want the report gone, they could always remove the record at the DB level

Reply:
@felipemontoya @mariajgrimaldi I have already removed delete options, I removed buttons and actions.